tasks/main.yml

```diff
@@ -1,3 +1,21 @@
+- name: "Install rsyslog-gnutls if using TLS (CentOS)"
+  yum:
+    name: "rsyslog-gnutls"
+    state: present
+  when: use_rsyslog_tls|bool == false and ansible_distribution == 'CentOS'
+  tags:
+    - rsyslog
+    - files
+
+- name: "Install rsyslog-gnutls if using TLS (Debian)"
+  apt:
+    name: "rsyslog-gnutls"
```

Comment:
Ok, will do. But next week. I need to get something out today. Thank you for the comments.